fs/src/lib.rs

```diff
@@ -20,23 +18,23 @@ use std::{
     path::{Path, PathBuf},
     sync::{Arc, RwLock},
 };
-use wascc_codec::core::CapabilityConfiguration;
 
 mod chunks;
 
 #[cfg(not(feature = "static_plugin"))]
 capability_provider!(FileSystemProvider, FileSystemProvider::new);
 
-const CAPABILITY_ID: &str = "wascc:blobstore";
+const CAPABILITY_ID: &str = "wasmcloud:blobstore";
 const SYSTEM_ACTOR: &str = "system";
 const FIRST_SEQ_NBR: u64 = 0;
 const VERSION: &str = env!("CARGO_PKG_VERSION");
-const REVISION: u32 = 3; // Increment for each crates publish
+const REVISION: u32 = 4; // Increment for each crates publish
```

Comment:
Revision and version are no longer needed for embedding in capability providers since we don't do descriptors.